src/index.js

```diff
@@ -1,8 +1,10 @@
-// import _ from 'lodash';
+import _ from 'lodash';// eslint-disable-line
```

Comment:
- [ ] `Required`
- [ ] Amazing job so far 👏 
- [ ] please enable the disabled linters because it defeats the purpose of installing linters to detect errors. 🥂